cluster/aws-cloudformation/jsonnet_to_json.sh

```diff
@@ -0,0 +1,3 @@
+# Assumes jsonnet is available on the path.
+jsonnet -s 10000000 --var USER="$USER" --var HOSTNAME="$HOSTNAME" --var TIMESTAMP="$(date)" ${@}
```

Comment:
I think we can remove the -s 10000000 which was fixed in https://github.com/google/jsonnet/issues/134